jobs/exportarInternaciones.ts

```diff
@@ -0,0 +1,161 @@
+                    CodNivelI: CodNivelI[informeDeIngreso.nivelInstruccion],
+                    NivelInst: informeDeIngreso.nivelInstruccion,
+                    CodSitLab: CodSitLab[informeDeIngreso.situacionLaboral],
+                    SitLab: informeDeIngreso.situacionLaboral,
+                    CodOcupac: (informeDeIngreso.ocupacionHabitual) ? informeDeIngreso.ocupacionHabitual.codigo : '',
+                    Ocupac: (informeDeIngreso.ocupacionHabitual) ? informeDeIngreso.ocupacionHabitual.nombre : '',
+                    CodHospPor: CodHospPor[informeDeIngreso.origen],
+                    HospPor: informeDeIngreso.origen,
+                    Origen: (informeDeIngreso.organizacionOrigen) ? informeDeIngreso.organizacionOrigen.nombre : '',
+                    FecIngreso: moment(informeDeIngreso.fechaIngreso).format('DD/MM/YYYY'),
+                    FecEgreso: (datosEgreso) ? moment(informeDeEgreso.fechaEgreso).format('DD/MM/YYYY') : '',
+                    EspecEgre: (informeDeIngreso.especialidades) ? informeDeIngreso.especialidades[0].term : '',
+                    CodEspecEgre: (informeDeIngreso.especialidades) ? informeDeIngreso.especialidades[0].conceptId : '',
+                    ServEgre: (mapEspecialidadServicio) ? mapEspecialidadServicio.descripcionDestino : '',
+                    CodServEgre: (mapEspecialidadServicio) ? mapEspecialidadServicio.codigoDestino : '',
+                    DiasTotEst: (datosEgreso) ? informeDeEgreso.diasDeEstada : '',
+                    CodEgresP: CodEgresP[(datosEgreso && informeDeEgreso.tipoEgreso) ? informeDeEgreso.tipoEgreso.nombre : 'Otro'],
+                    EgresP: (datosEgreso && informeDeEgreso.tipoEgreso) ? informeDeEgreso.tipoEgreso.nombre : '',
+                    Lugar_Trasl: (datosEgreso && informeDeEgreso.UnidadOrganizativaDestino) ? informeDeEgreso.UnidadOrganizativaDestino.nombre : '',
+                    CodDiagPr: (datosEgreso && informeDeEgreso.diagnosticoPrincipal) ? informeDeEgreso.diagnosticoPrincipal.codigo : '',
+                    OtDiag1: (datosEgreso && informeDeEgreso.otrasCircunstancias) ? informeDeEgreso.otrasCircunstancias.nombre : '',
+                    CodCauExtT: (datosEgreso && informeDeEgreso.causaExterna.producidaPor !== null) ? CodCauExtT[informeDeEgreso.causaExterna.producidaPor.nombre] : CodCauExtT['Se ignora'],
+                    CauExtT: (datosEgreso && informeDeEgreso.causaExterna.producidaPor !== null) ? informeDeEgreso.causaExterna.producidaPor.id : 'Se ignora',
+                    CodCauExtL: (datosEgreso && informeDeEgreso.causaExterna.lugar !== null) ? CodCauExtL[informeDeEgreso.causaExterna.lugar.nombre] : CodCauExtL['Se ignora'],
+                    CauExtL: (datosEgreso && informeDeEgreso.causaExterna.lugar !== null) ? informeDeEgreso.causaExterna.lugar.nombre : 'Se ignora'
+                };
+                if (datosEgreso && informeDeEgreso.nacimientos && informeDeEgreso.nacimientos[0].condicionAlNacer) {
```

Comment:
```suggestion
                if (datosEgreso && informeDeEgreso.nacimientos && informeDeEgreso.nacimientos.length && informeDeEgreso.nacimientos[0].condicionAlNacer) {
```
Puede ser vacío y daría error

Reply:
a que campo te referis @lucas-gonzalez ??